cmd/rekor-cli/app/get.go

```diff
@@ -150,13 +150,16 @@ func parseEntry(uuid string, e models.LogEntryAnon) (interface{}, error) {
 	}
 
 	obj := getCmdOutput{
-		Attestation:     string(e.Attestation.Data),
-		AttestationType: e.Attestation.MediaType,
-		Body:            eimpl,
-		UUID:            uuid,
-		IntegratedTime:  *e.IntegratedTime,
-		LogIndex:        int(*e.LogIndex),
-		LogID:           *e.LogID,
+		Body:           eimpl,
+		UUID:           uuid,
+		IntegratedTime: *e.IntegratedTime,
+		LogIndex:       int(*e.LogIndex),
+		LogID:          *e.LogID,
+	}
+
+	if viper.GetBool("enable_attestation_storage") {
```

Comment:
I dont think this command line option would be given in the CLI; I think the best way to do this would be to see if `e.Attestation != nil` in the response

Reply:
@bobcallaway Thanks, good point, I agree that is more direct and I just updated the code and re-pushed. I was originally using some of the `api/entries.go` [code](https://github.com/sigstore/rekor/blob/070f83e924cc837178a09b2b4d1089562c8f23da/pkg/api/entries.go#L96) as an example.